### PR TITLE
Minor bug fix and an integration test fix

### PR DESCRIPTION
**Pull request by ashitsalesforce**

### CODE REVIEW

File: src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java [hunk 1]
```diff
@@ -214,11 +214,20 @@ public void testDescribeSObjects() throws Exception {
...
         for (String objectType : client.getDescribeGlobalResults().keySet()){
-            DescribeSObjectResult describeResult = client.describeSObject(objectType);
-            numDescribes++;
-            assertNotNull(describeResult);
-            assertEquals(objectType, describeResult.getName());
-            assertEquals(numDescribes, client.getEntityDescribeMap().size());
+        	try {
+        		DescribeSObjectResult describeResult = client.describeSObject(objectType);
+        		numDescribes++;
+                assertNotNull(describeResult);
```
Comment (jjangsfdc) [inline src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java:220]: Minor, indentation.

File: src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java [hunk 2]
```diff
@@ -214,11 +214,20 @@ public void testDescribeSObjects() throws Exception {
...
+                assertEquals(objectType, describeResult.getName());
+                assertEquals(numDescribes, client.getEntityDescribeMap().size());
+        	} catch (Exception ex) {
+        		if (ex.getMessage().contains("jsonNot")) {
+        			System.out.println("PartnerClient.testDescribeSObjects: Unable to call describeSObject for " + objectType);
```
Comment (jjangsfdc) [inline src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java:225]: I think it's better to throw an exception and include the custom message.

File: release/mac/dataloader.command [hunk 3]
```diff
@@ -25,6 +25,6 @@ then
     echo "Zulu OpenJDK 11 is not installed.  Download Zulu OpenJDK 11 for macOS here: https://www.azul.com/downloads/zulu/zulu-mac/"
 else
     echo "$JAVA_HOME"
-    cd DATALODER_WORK_DIRECTORY_PLACEHOLDER   #change to your own customized directory
-    java -XstartOnFirstThread -jar dataloader-@@FULL_VERSION@@-uber.jar salesforce.config.dir=DATALODER_WORK_DIRECTORY_PLACEHOLDER/configs
+    cd DATALOADER_WORK_DIRECTORY_PLACEHOLDER
```
Comment (jjangsfdc) [inline release/mac/dataloader.command:28]: Good catch

**Review by jjangsfdc — APPROVED** [review]
Other then few minor ones, looks good.

File: src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java (hunk 1 above)
Comment (ashitsalesforce) [reply src/test/java/com/salesforce/dataloader/client/PartnerClientTest.java:220]: good catch. I need to configure my IDE to treat a tab as 4 space characters.

**Pull request merged**